src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,47 @@
+.calendar {
+  padding: 20px 10px;
+  margin: 0 auto;
+  width: $full-width;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+
+  &__day {
+    width: $cell-size;
+    height: $cell-size;
+
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    background-color: $cell-color;
+    border: 1px solid #000;
+    transition: background-color 500ms, transform 500ms;
+
+    &:hover {
+      cursor: pointer;
```

Comment:
Why alter `cursor` on hover?

Reply:
Thanks for you advice about the spelling checker!) I will install it right now!)

I imagined that this can be an application and you can open your plans for the day you click on, but yeah, I will delete the cursor.)